### ISSUE TITLE
Update peer dependency to support @azure/msal-react 2.x

### ISSUE THREAD

**Issue by ottD** [Mimetis/msal-react-tester#17]
Version 2.0.0 of the Microsoft Authentication Library for React ([msal-react](https://github.com/AzureAD/microsoft-authentication-library-for-js/releases/tag/msal-react-v2.0.0 )) has been release a few days ago. It would be great if we could keep using msal-react-tester with the latest version of @azure/msal-react. 

```
Found: @azure/msal-react@2.0.0
node_modules/@azure/msal-react
  @azure/msal-react@"^2.0.0" from the root project

Could not resolve dependency:
peer @azure/msal-react@"^1.5.4" from msal-react-tester@0.2.2
node_modules/msal-react-tester
  dev msal-react-tester@"*" from the root project
```

**Comment by Mimetis** [comment]
Seems the version `2.0.0` is broking everything unfortunately :(

<img width="664" alt="image" src="https://github.com/Mimetis/msal-react-tester/assets/4592555/074f03fc-b8ed-45cd-a24d-108f2c51642c">

I've updated the package to allow any version of `msal-react`, but I guess you should stay on `1.5.10` until the issue is resolved on their side

**Comment by bsides** [comment]
@Mimetis Could you explain the message above a little more? I don't understand, if MSAL is working fine, why should they fix something on their side?

**Comment by Mimetis** [comment]
I don't know
- With `create-react-app`, it just does not work at all.
- With `Vite`, I had to put `"type"="module"` in the `/node_modules/@azure/msal-common/package.json` file 

I've published the version [0.3.1](https://www.npmjs.com/package/msal-react-tester), give it a try and let me know how it's working (or not) for you

**Comment by Mimetis** [comment]
See more here : https://github.com/AzureAD/microsoft-authentication-library-for-js/issues/6317

**Comment by Mimetis** [comment]
The issue seems to be fixed with version `2.01` of `msal-react`

That being said, it does not work anymore with `create-react-app`.
As it's deprecated, I will remove the sample and try to make a sample with `NextJS`

**Comment by ottD** [comment]
@Mimetis Many thanks for the quick response and fix! I'm using `Vite` and it is working perfectly.
